string-manipulation-3/string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,70 @@
+/* exported titleCase */
+function titleCase(title) {
+  const titleArray = title.toLowerCase().split(' ');
+  console.log(titleArray);
```

Comment:
`console.log` is only for debugging / testing purposes, so they should be removed whenever submitting an exercise! Unless we specifically ask for you to include them

Reply:
Sorry forgot to remove this when I was debugging.